flake.nix

```diff
@@ -0,0 +1,35 @@
+{
+  inputs = {
+    nixpkgs.url = "github:nixos/nixpkgs/nixos-unstable";
```

Comment:
Do not pin non-nixos packages to nixos-unstable branch.
Prefer nixpkgs-unstable or master.

Reply:
Done